test/functional/apps/dashboard/panel_context_menu.ts

```diff
@@ -114,18 +117,29 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
 
         const panelCount = await PageObjects.dashboard.getPanelCount();
         expect(panelCount).to.be(0);
+        // need to find the correct save
+        await PageObjects.dashboard.saveDashboard(dashboardName);
       });
     });
 
     describe('saved search object edit menu', () => {
       const searchName = 'my search';
 
       before(async () => {
+        await opensearchDashboardsServer.uiSettings.replace({
+          'discover:v2': false,
+        });
+        await browser.refresh();
         await PageObjects.header.clickDiscover();
         await PageObjects.discover.clickNewSearchButton();
         await dashboardVisualizations.createSavedSearch({ name: searchName, fields: ['bytes'] });
         await PageObjects.header.waitUntilLoadingHasFinished();
         await PageObjects.header.clickDashboard();
+        // Have to add steps to actually click on the dashboard; since added browser.refresh() will make
+        // clickDashboard() to only land on the dashboard listing page
+        // We need to add browser.refresh() so clickDiscover() lands correctly on the legacy discover page
```

Comment:
Not for right now but when we remove Legacy discover in 2.11, is it easy enough to migrate these to Discover 2.0?